core/shared/util.ts

```diff
@@ -95,4 +96,15 @@ export class Util {
       muted = isPassword;
     });
   }
+
+  // --- JELLYFISH UTIL --- //
+  static readNetwork(): Network {
```

Comment:
Use this for signature verification (`crypto.ts`)? 😉

Reply:
would be a good idea :D will adapt